stdlib/public/core/CMakeLists.txt

```diff
@@ -310,6 +315,10 @@ list(APPEND swift_stdlib_compile_flags "-enable-experimental-feature" "Freestand
 list(APPEND swift_stdlib_compile_flags "-enable-experimental-feature" "Extern")
 list(APPEND swift_stdlib_compile_flags "-enable-experimental-feature" "BitwiseCopyable")
 
+list(APPEND swift_stdlib_compile_flags "-external-plugin-path"
+  "${CMAKE_BINARY_DIR}/lib/swift/host/plugins#${CMAKE_BINARY_DIR}/bin/swift-plugin-server")
```

Comment:
It seems this change broke cross-host builds: https://ci.swift.org/job/oss-swift-package-macos/2718/console
```
sandbox-exec: execvp() of '/Users/ec2-user/jenkins/workspace/oss-swift-package-macos/build/buildbot_osx/swift-macosx-arm64/bin/swift-plugin-server' failed: Bad CPU type in executable
```

I guess we need to use swift-plugin-server under `SWIFT_NATIVE_SWIFT_TOOLS_PATH` if it's specified.

Reply:
thanks, working on it

Reply:
https://github.com/apple/swift/pull/71814

Is there a way to validate the cross-host builds from the PR?

Reply:
Thank you!